test/common_methods_invocations.py

```diff
@@ -810,7 +810,7 @@ def maybe_non_contig(tensor):
         elif isinstance(arg, torch.Tensor):
             if arg.dtype == torch.float:
                 arg = arg.double()
-            v = maybe_non_contig(arg).detach()
+            v = maybe_non_contig(arg).detach().clone()
```

Comment:
The original semantics allows size/storage changes to `v` after this line, and we preserve this behavior by cloning after `detach()`.